src/app/modules/core/advertisement/mock-advertisemente/mock-advertisement.ts

```diff
@@ -0,0 +1,105 @@
+import { Category, PhoneType, Role, Status, Toilet } from '@shared/enum/enums'
+import { IReviews } from '@shared/models/advertisement'
+import { IProperty } from '@shared/models/property'
+import { IUser } from '@shared/models/users'
+
+export interface IMockAdvertisement {
+  id: number
+  landlord: IUser
+  property: IProperty
+  reviews: IReviews
```

Comment:
Qui forse è meglio mettere le parentesi quadre perché è un array di IReviews

Reply:
> Qui forse è meglio mettere le parentesi quadre perché è un array di IReviews

@Ste91wq ha ragione e torto :D nel senso, che `reviews` dovrebbe essere di tipo array of `IReview` and `IReview` dovrebbe mappare una singola review.

Reply:
si, hai ragione